lib/astarte_data_access/entities/realm.ex

```diff
@@ -0,0 +1,912 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2024 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Astarte.DataAccess.Realm do
+  require Logger
+
+  alias Astarte.DataAccess.CSystem
+  alias Astarte.DataAccess.XandraUtils
+
+  @default_realm_schema_version 10
+
+  def create_realm(
+        realm_name,
+        replication,
```

Comment:
old code used to have 
```
@default_replication_factor 1
```
now it seems nowhere to be found? there is no default or has it been moved to another file?